dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/FastSynchHashTable.java

```diff
@@ -0,0 +1,336 @@
+/*******************************************************************************
+ * Copyright (c) 2005, 2006 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.netty.internal.tcp;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+
+/**
+ * FastSyncHashTable.
+ * 
+ * This is a table implementation that does very little besides basic table operations.
+ * In making it fast, it does not maintain counts of items in the table, cannot grow or change in size once instantiated,
+ * and therefore has no global tablewide locks. This locks at the bucket level, providing for smaller granularity of
+ * of locking and therefore faster access of data. The main table is a 2-d array allowing for some variation. Both the
+ * hashCode() and equals() methods are used to locate and compare keys to find entries in the table.
+ * 
+ * Copied from {@link com.ibm.ws.bytebuffer.internal.FastSynchHashTable}
+ * TODO: get rid of this when we have a common WsByteBuffer bundle
```

Comment:
Convert TODO to issue?

Reply:
you've already fixed this with https://github.com/OpenLiberty/open-liberty/pull/17940 - I'll update this PR to use your new project